controllers/system/system_controller.go

```diff
@@ -1102,6 +1102,19 @@ func (r *SystemReconciler) ReconcileRequired(instance *starlingxv1.System, spec
 		return err, false
 	}
 
+	var res starlingxv1.CertificateList
```

Comment:
Expect a new method to wrap the new codes, and a unit test against it

Reply:
Need to add description here to explain why not add the other certs in the map to the current list

Reply:
I have updated accordingly